### ISSUE TITLE
Notification of nsca for `nagiostopic/spu.sources:08/SERVICE_STATUS' FAILED or TIMED OUT

### ISSUE THREAD

**Issue by adri641** [jpmens/mqttwarn#437]
Hi,
i have set mqttwarn to take some values via mosquitto from an application and then send to my nagios infrastructure via nsca.
I am receiving this error when it tries to send the data:
2020-07-20 16:35:40,517 WARNING  [mqttwarn.services.nsca   ] Cannot notify to NSCA host `X.X.X.X': must be str, not bytes
2020-07-20 16:35:40,517 WARNING  [mqttwarn.core            ] Notification of nsca for `nagiostopic/spu.sources:08/SERVICE_STATUS' FAILED or TIMED OUT

My configuration for the nsca is the following:

[config:nsca]
nsca_host = 'X.X.X.X'
targets = {
   'test_prismon'    :  [ 'PRISMON01',          'Current temp via MQTT' ],
  }

[check_channel_nagios/test_prismon]
targets = nsca:test_prismon
; OK = 0, WARNING = 1, CRITICAL = 2, UNKNOWN = 3
format = Warning: Service 08 {_dthhmmss}  Status: {payload}
topic   = nagiostopic/spu.sources:08/SERVICE_STATUS
filter = check_channelstatus()

This is the method i use for filtering

def check_channelstatus(topic, message):
    OK = "0"
    WARNING = "1"
    CRITICAL = "2"
    UNKNOWN = "3"
    temp = float(message)
    if temp == 0:
        return OK
    if temp > 0:
        return CRITICAL
    if temp < 0:
        return CRITICAL

    return UNKNOWN

Basically what i need to do is send an alarm based on the value of the payload in the broker topic.

If it is 0 it means that is ok, 1 and -1 is not ok, so "Critical".

Is it a problem of maybe my nagios system or there is something wrong in the code/configuration of mqttwarn?
